# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium/sandbox/win/src/win2k_threadpool.cc
# Commit: 370c40e49230
# Full Hash: 370c40e4923073a87845b57c1f4851bacd9663da
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-12 21:42:35
# Regressor Bug: 1552160
# File Overlap Count: 2
# Description:
#   Bug 1552160 Part 1: Roll-up of chromium sandbox update and mozilla patches to get a running browser. r=jld,aklotz,tjr,bobowen
#   
#   This updates security/sandbox/chromium/ files to chromium commit 84108231f6e6e0772fb9a4643679ce76aa771e67.
#   
#   Existing and new patches applied from security/sandbox/chromium-shim/patches/with_update/ to give a compiling and mostly working browser.
# ==============================================================================

diff -r bd5ecd72f76c -r 370c40e49230 security/sandbox/chromium/sandbox/win/src/win2k_threadpool.cc
--- a/security/sandbox/chromium/sandbox/win/src/win2k_threadpool.cc	Wed Jun 12 21:55:59 2019 +1200
+++ b/security/sandbox/chromium/sandbox/win/src/win2k_threadpool.cc	Wed Jun 12 11:10:48 2019 +0100
@@ -14,13 +14,14 @@
   ::InitializeCriticalSection(&lock_);
 }
 
-bool Win2kThreadPool::RegisterWait(const void* cookie, HANDLE waitable_object,
+bool Win2kThreadPool::RegisterWait(const void* cookie,
+                                   HANDLE waitable_object,
                                    CrossCallIPCCallback callback,
                                    void* context) {
   if (0 == cookie) {
     return false;
   }
-  HANDLE pool_object = NULL;
+  HANDLE pool_object = nullptr;
   // create a wait for a kernel object, with no timeout
   if (!::RegisterWaitForSingleObject(&pool_object, waitable_object, callback,
                                      context, INFINITE, WT_EXECUTEDEFAULT)) {